test/test_notebook.py

```diff
@@ -1,5 +1,7 @@
+import itertools
 import sys
 import os
+import random

```

Comment:
random is not used, which can be removed?

Reply:
same w/ itertools

Reply:
Added by accident.